pyadr/core.py

```diff
@@ -1,6 +1,7 @@
 import re
 import shutil
 from pathlib import Path
+from tkinter import W
```

Comment:
This does not seem to be used. Or is it?

Reply:
Sorry for the delay. I removed it.